core/src/main/resources/hudson/PluginManager/_table.js

```diff
@@ -9,30 +9,42 @@ function checkPluginsWithoutWarnings() {
 }
 
 Behaviour.specify("#filter-box", '_table', 0, function(e) {
-      function applyFilter() {
-          var filter = e.value.toLowerCase();
-          ["TR.plugin","TR.plugin-category"].each(function(clz) {
-            var encountered = {};
```

Comment:
`encountered` and `TR.plugin-category` have been obsolete since https://github.com/jenkinsci/jenkins/pull/4534.